src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerTypeSymbol.cs

```diff
@@ -186,5 +200,31 @@ internal static bool RefKindEquals(TypeCompareKind compareKind, RefKind refKind1
         /// </summary>
         internal static RefKind GetRefKindForHashCode(RefKind refKind)
             => refKind == RefKind.None ? RefKind.None : RefKind.Ref;
+
+        /// <summary>
+        /// Return true if the given type is valid as a calling convention modifier type.
+        /// </summary>
+        internal static bool IsCallingConventionModifier(NamedTypeSymbol modifierType)
```

Comment:
>IsCallingConventionModifier [](start = 29, length = 27)

Was this function simply moved from FunctionPointerMethodSymbol,, or were there any changes in implementation?

Reply:
The only change was that this method previously took a `CustomModifier`, now it takes the symbol of the modifier so it can be reused in other locations.

---
In reply to: [463854327](https://github.com/dotnet/roslyn/pull/46467#discussion_r463854327) [](ancestors = 463854327)

Reply:
Oh, and I reordered the `Arity` test to come before the `Name != "CallConv"` from Jared's feedback on my previous PR.

---
In reply to: [463861562](https://github.com/dotnet/roslyn/pull/46467#discussion_r463861562) [](ancestors = 463861562,463854327)